### PR TITLE
Adds allow_service parameter to bigip_selfip

### PR DESCRIPTION
**Pull request by caphrim007**
##### ISSUE TYPE
- Feature Pull Request
##### COMPONENT NAME

network/f5/bigip_selfip.py
##### ANSIBLE VERSION

```
ansible 2.2.0
```
##### SUMMARY

This parameter can be used to open up access to (among other things)
the mgmt address of a BIG-IP. It is necessary for configuring bigips
in an HA configuration.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @caphrim007. This module is going into community review.

We notice that you are one of the maintainers (@mhite, @caphrim007) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

File: network/f5/bigip_selfip.py [hunk 1]
```diff
@@ -90,9 +96,76 @@
...
+          - tcp:0
+  delegate_to: localhost
 '''
 
 RETURN = '''
+allow_service:
+    description: Services that allowed via this Self IP
+    returned: changed
+    type: list
+    sampel: ['igmp:0','tcp:22','udp:53']
```
Comment (mhite) [inline network/f5/bigip_selfip.py]: sampel -> sample

**Comment by caphrim007** [comment]
@mhite fixed

**Comment by mhite** [comment]
shipit

**Comment by gregdek** [comment]
Thanks again to @caphrim007 for this PR, and thanks @mhite, @caphrim007 for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request merged**